usr/lib/jargonaut/jargonaut.py

```diff
@@ -447,7 +447,7 @@ def repl(match):
             if not self.is_window_focused():
                 self.tray.set_icon_name("jargonaut-status-msg-symbolic")
                 title = _("Message from %s") % message.nick
```

Comment:
I think this needs to be `_message.nick`.

Reply:
Thanks! Fixed.